src/ride/ride.c

```diff
@@ -4606,6 +4623,11 @@ int ride_is_valid_for_test(int rideIndex, int goingToBeOpen, int isApplying)
 	rct_xy_element trackElement, problematicTrackElement;
 
 	ride = GET_RIDE(rideIndex);
```

Comment:
Done (but github won't hide this comment, as it was made one line above the change)